### PR TITLE
Pre-release merge for arXiv docs v0.2.8

### PR DESCRIPTION
**Pull request by mhl10**

### CODE REVIEW

File: help/submit_tex.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ TeX-based supplemntal material should be included in the main document root dire
   ./supplement.bbl
 ```
```
Comment (eawoods) [inline help/submit_tex.md]: Wasn't part of the original diff, but by chance I wanted a bit more context and opened the area above this diff. There's a typo on line 135, `supplemental`.

File: help/mathjax.md [hunk 2]
```diff
@@ -59,20 +57,20 @@ link to set a cookie:
...
-shortcut for $\\mathbf{R}\^2$, then MathJax will not be able to render
-this expression. Submitters should use only standard $\\LaTeX$ and
-AMS-$\\LaTeX$ macros in paper titles and abstracts.
+user-defined macros. For example, if an abstract contains <span class="mathjax">$\\rtwo$</span> as a
+shortcut for <span class="mathjax">$\\mathbf{R}\^2$</span>, then MathJax will not be able to render
+this expression. Submitters should use only standard <span class="mathjax">La$\\TeX$</span> and
+AMS-La<span class="mathjax">$\\TeX$</span> macros in paper titles and abstracts.
 
 Also, since '\<' is a reserved character in HTML, special care must be
 taken in its use. Usually just adding a space after it will suffice for
```
Comment (eawoods) [inline help/mathjax.md]: In viewing on beta, the entity reference that I think is supposed to display as `&lt;` is coming up as just `<` (line 78)

File: help/submit_tex.md (hunk 1 above)
Comment (mhl10) [reply help/submit_tex.md]: Fixed (and another minor typo I noticed)!

File: help/mathjax.md (hunk 2 above)
Comment (mhl10) [reply help/mathjax.md]: Fixed

**Pull request merged**